languages/ru-RU.js

```diff
@@ -0,0 +1,196 @@
+				].join(' ')),
+				'Пожалуйста, создайте issue на <https://github.com/dirigeants/klasa>, если найдёте какие-либо ошибки.'
+			],
+			COMMAND_INVITE_DESCRIPTION: 'Отображает ссылку для добавления бота на сервер',
+			COMMAND_INFO: [
+				"Klasa это 'plug-and-play' фреймворк, работающий поверх библиотеки Discord.js.",
+				'Большая часть кода разделена на модули, что позволяет разработчикам изменять Klasa под свои нужды.',
+				'',
+				'Некоторые особенности Klasa:',
+				'• 🐇💨 Быстрая загрузка с поддержкой ES2017 (`async`/`await`)',
+				'• 🎚🎛 Расширяемые настройки для каждого клиента/пользователя/сервера',
+				'• 💬 Настраиваемая система команд с автоматическим разрешением параметров и возможностью загружать/перезагружать команды на лету',
+				'• 👀 "Monitors", отслеживающие сообщения и их редактирование (фильтры бранных слов, защита от спама)',
+				'• ⛔ "Inhibitors", предотвращающие выполнение команды на основе заданных условий (права доступа, чёрные списки)',
+				'• 🗄 "Providers", упрощающие использование базы данных по вашему выбору',
+				'• ✅ "Finalizers", выполняющиеся после успешного выполнения команд (логирование, сбор статистики, очистка ответов)',
+				'• ➕ "Extendables", расширяющие существующие классы Klasa и Discord.js новыми методами, геттерами/сеттерами и статическими свойствами',
+				'• 🌐 "Languages", позволяющие локализировать ответы вашего бота',
+				'• ⏲ "Tasks", которые могут быть запланированы для выполнения в будущем с возможностью повтора',
+				'',
+				'Мы надеемся быть на 100% настраиваемым фреймворком, который бы устраивал всех разработчиков. Мы часто выпускаем обновления и исправляем ошибки по мере возможности.',
+				'Если вы заинтересованы, вы можете найти нас на https://klasa.js.org'
+			],
+			COMMAND_INFO_DESCRIPTION: 'Предоставляет информацию о боте.',
+			COMMAND_HELP_DESCRIPTION: 'Отображает список доступных вам команд или справку о конкретной команде.',
+			COMMAND_HELP_NO_EXTENDED: 'Расширенная справка отсутствует.',
+			COMMAND_HELP_DM: '📥 | Список доступных вам команд отправлен вам в личные сообщения.',
+			COMMAND_HELP_NODM: '❌ | Я не могу отправить вам список доступных команд, так как вы отключили личные сообщения.',
+			COMMAND_HELP_USAGE: (usage) => `Формат :: ${usage}`,
+			COMMAND_HELP_EXTENDED: 'Расширенная справка ::',
+			COMMAND_ENABLE: (type, name) => `+ Успешно активировано ${type}: ${name}`,
+			COMMAND_ENABLE_DESCRIPTION: 'Активирует или временно активирует команду/ингибитор/монитор/финализатор. Состояние по-умолчанию буде восстановлено после перезагрузки.',
+			COMMAND_DISABLE: (type, name) => `+ Успешно деактивировано ${type}: ${name}`,
+			COMMAND_DISABLE_DESCRIPTION: 'Деактивирует или временно деактивирует команду/ингибитор/монитор/финализатор/событие. Состояние по-умолчанию буде восстановлено после перезагрузки.',
+			COMMAND_DISABLE_WARN: 'Скорее всего вам не стоит деактивировать это, так как вы не сможете использовать какую-либо команду чтобы активировать это снова.',
```

Comment:
A comma before "чтобы" is needed.
```suggestion
			COMMAND_DISABLE_WARN: 'Скорее всего вам не стоит деактивировать это, так как вы не сможете использовать какую-либо команду, чтобы активировать это снова.',
```